uiowa/uiowa
# [Menu block] Horizontal menu with centered alignment

Resolves https://github.com/uiowa/uiowa/issues/7703


# How to test

```
ddev blt frontend && ddev blt ds --site=tippie.uiowa.edu  && ddev drush @tippie.local uli  /faculty-research
```

- Edit layout and edit the menu block
- Select "Center" from "Aligment" layout builder style at bottom of menu block edit
- Confirm that the alignment option only appears if "Horizontal" is selected
- Save page and confirm that only top level menu is centered and dropdowns are not centered
- Check mobile view and confirm everything is still left aligned
- Test with "Vertical" alignment and confirm that everything looks as expected
- **Question:** Should we unset the "Center" value if you switch back to "Vertical" - we do this in other blocks, but in this case having the center alignment class on the vertical navigation does not change the appearance of the menu.  Reference: https://github.com/uiowa/uiowa/blob/main/docroot/modules/custom/layout_builder_custom/js/image-block-form.js#L14
